### PR TITLE
Publish preparation

### LINKED ISSUE #10: Publish Preparation

### PR DESCRIPTION
**Pull request by BoghianRobert**
closes #10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +36,21 @@
     "metro-react-native-babel-preset": "^0.70.3",
     "react-test-renderer": "18.0.0"
   },
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*",
```
Comment (lgnstn) [inline package.json:41]: I think the "dependencies" section should be removed, right?

**Review by lgnstn — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
+  "license": "ISC",
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint ."
   },
-  "dependencies": {
-    "@react-navigation/native": "6.0.12",
```
Comment (lgnstn) [inline package.json:13]: This one is not present in the list of "peerDependencies". Is it ok?

Comment (lgnstn) [reply package.json:13]: Same thing with prop-types.

**Review by lgnstn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (BoghianRobert) [reply package.json:13]: Yes, the library doesn't use the navigation dependency anymore. I also added the prop-types to peerDependencies

**Review by lgnstn — APPROVED** [review]

**Pull request merged**